src/renderers/dom/client/utils/createObjectWithProto.js

```diff
@@ -0,0 +1,33 @@
+/**
```

Comment:
Functionally speaking, there's probably a small benefit to using `Object.create` if available, because the created object may look better in the dev inspector/debugger, but otherwise there is very little difference.